examples/0-counter-part.elm

```diff
@@ -21,13 +19,13 @@ main =
 
 
 type alias Model =
-  { counter : Indexed Counter.Model 
+  { counters : Counters
   }
 
 
 init : (Model, Cmd Msg)
 init =
-  ( { counter = Dict.empty }
+  ( { counters = .empty Counters.all }
```

Comment:
Unfortunatly, this yields a very obscure error message, when writing `Counters.all.empty`, see [#520](https://github.com/elm-lang/elm-compiler/issues/520)